lib/galaxy/queue_worker.py

```diff
@@ -18,6 +19,21 @@
 log = logging.getLogger(__name__)
 
 
+def queue_async_task(app, task, kwargs={}):
+    log.info("Queuing async task %s." % task)
```

Comment:
Yes, and, yes.

Thanks @mvdbeek, I'll add better (some!) documentation and I like the new name for the method, will update.